frontend/cypress/support/index.ts

```diff
@@ -1,8 +1,11 @@
+import './commands';
+
 declare namespace Cypress
 {
 	interface Chainable<Subject>
 	{
 		login() : Chainable<Subject>
+		logout() : Chainable<Subject>
 	}
 }
-Cypress.LocalStorage.clear();
+Cypress;
```

Comment:
What is Cypress; at the end doing?

Reply:
This is old I only have import './command';
in my index.ts